event-publisher/src/main/java/uk/gov/ons/ctp/common/event/model/NewCasePayloadContent.java

```diff
@@ -1,18 +1,38 @@
 package uk.gov.ons.ctp.common.event.model;
 
+import java.util.Map;
 import java.util.UUID;
 import lombok.AllArgsConstructor;
+import lombok.Builder;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
+@Builder
 public class NewCasePayloadContent implements EventPayload {
 
+  // TODO : FLEXIBLE CASE
```

Comment:
remove TODO ?

Reply:
I asked Peter to add that. Same comment is elsewhere. I want to identify for future work the areas where we have parked what to do about handling flexible sample/metadata.